scripts/setup/linux/ubuntu/xenial/install_prereqs

```diff
@@ -77,6 +77,7 @@ libnetcdf-cxx-legacy-dev
 libnetcdf-dev
 libnlopt-dev
 libogg-dev
+libpcl-dev
```

Comment:
Add a comment that this is not a drake dependency, but a shambhala example dependency.

Reply:
Since I can't put a comment in the heredoc I've moved the pcl dependency out into a further apt-get invocation with 83efb405c4b2a8e4371e4e3e683396e6b00abe23